src/lib/plugin-loader.js

```diff
@@ -55,19 +56,22 @@ function load_plugins(config, plugin_configs, params, sanity_check) {
     }
 
     if (plugin === null) {
+      Logger.logger.error({content: p}, 'plugin not found. try npm install verdaccio-@{content}');
       throw Error('"' + p + '" plugin not found\ntry "npm install verdaccio-' + p + '"');
     }
 
     if (typeof(plugin) !== 'function') {
+      Logger.logger.error({content: p}, '@{content} doesn\'t look like a valid plugin');
       throw Error('"' + p + '" doesn\'t look like a valid plugin');
     }
 
     plugin = plugin(plugin_configs[p], params);
 
     if (plugin === null || !sanity_check(plugin)) {
+      Logger.logger.error({content: p}, '@{content} doesn\'t look like a valid plugin');
       throw Error('"' + p + '" doesn\'t look like a valid plugin');
     }
-
+    Logger.logger.warn({content: p}, 'Plugin successfully loaded: @{content}');
```

Comment:
One more thing, perhaps `warn` is too aggressive. Users might think there is an error. Perhaps we should use `info` instead?

Reply:
Yes,  I've updated the PR.